TODOS.md

```diff
@@ -0,0 +1,7 @@
+Last updated: Tue Jan 15 2019 - 11:27:58 AM
+
+# TODO:
+
+- [packages/alfa-css/src/properties/text-decoration/color/property.ts:20](https://github.com/siteimprove/alfa/blob/todo-list-fix/packages/alfa-css/src/properties/text-decoration/color/property.ts#L20): Should be currentColor when supported in colorGrammar
```

Comment:
Links in GitHub markdown can be relative from the repository root so `https://github.com/siteimprove/alfa/blob/todo-list-fix/` can be left out.